repo/content/committed_read_manager.go

```diff
@@ -643,8 +643,9 @@ func NewSharedManager(ctx context.Context, st blob.Storage, prov format.Provider
 	sm.log = sm.namedLogger("shared-manager")
 
 	caching = caching.CloneOrDefault()
+	writesOnIndexLoad := opts.AllowCleanupWritesOnIndexLoad
 
-	if err := sm.setupCachesAndIndexManagers(ctx, caching, mr); err != nil {
+	if err := sm.setupCachesAndIndexManagers(ctx, caching, mr, writesOnIndexLoad); err != nil {
```

Comment:
This wires `content.ManagerOptions.AllowCleanupWritesOnIndexLoad` to `epoch.Manager.allowCleanupWritesOnIndexLoad`